gen/generator-config.json

```diff
@@ -5,7 +5,7 @@
   "apiPackage": "Api",
   "srcBasePath": "MuxPhp",
   "invokerPackage": "MuxPhp",
-  "artifactVersion": "3.18.0",
+  "artifactVersion": "3.19.0",
```

Comment:
Just curious, why is PHP one minor version ahead?

Reply:
We had to release something PHP-specific in the past